docs/install.md

```diff
@@ -384,14 +384,12 @@ This step is optional but makes it much easier to connect to and administer the
 Host app
   Hostname fsrrszf5qw7z2kjh.onion
   User <ssh_user>
-  ProxyCommand connect -R remote -5 -S localhost:9050 %h %p
 Host mon
   Hostname yt4j52ajfvbmvtc7.onion
   User <ssh_user>
-  ProxyCommand connect -R remote -5 -S localhost:9050 %h %p
 ```
 
-Now you can simply use `ssh app` and `ssh mon` to connect to each server, and it will be stored in the Tails Dotfiles persistence. You might see the message "Enter SOCKS5 password for amnesia@localhost" — this can be ignored, just always press Enter.
+Now you can simply use `ssh app` and `ssh mon` to connect to each server, and it will be stored in the Tails Dotfiles persistence.
```

Comment:
Using torify on Tails spews out a bunch of errors (below) when using torify even though it still torify still works. So not sure what the best course of action is.

```
12:22:01 libtorsocks(6492): WARNING: The symbol res_query() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_query() with the reported error: Not Found
12:22:01 libtorsocks(6492): WARNING: The symbol res_search() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_search() with the reported error: Not Found
12:22:01 libtorsocks(6492): WARNING: The symbol res_send() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_send() with the reported error: Not Found
12:22:01 libtorsocks(6492): WARNING: The symbol res_querydomain() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_querydomain() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_query() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_query() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_search() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_search() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_send() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_send() with the reported error: Not Found
12:22:01 libtorsocks(6493): WARNING: The symbol res_querydomain() was not found in any shared library with the reported error: Not Found!
  Also, we failed to find the symbol __res_querydomain() with the reported error: Not Found
```